doc/tutorial.md

```diff
@@ -0,0 +1,175 @@
+  under current user folder for Windows),
+  an example is as following:
+
+  ```bash
+  datacenters:
+  - name: production
+    username: paddlepaddle
+    password: paddlecloud
+    endpoint: http://production.paddlecloud.com
+  - name: experimentation
+    username: paddlepaddle
+    password: paddlecloud
+    endpoint: http://experimentation.paddlecloud.com
+  current-datacenter: production
+  ```
+
+  Supports you have two data-center's access, one for `production` and another one for
+  `experimentation`, you can choose one data-center by the `current-datacenter` field
+  for the current one.
+
+  `username`, `password` and `endpoint` is your secret for the data-center, you will
+  received an email including your secret information send by the administrator.
+
+After finish the two steps above, execute `paddlectl` command will print the usage:
+
+```bash
+> paddlectl
+Usage: paddlecloud <flags> <subcommand> <subcommand args>
+
+Subcommands:
+  commands         list all command names
+  delete           Delete the specify resource.
+  file             Simple file operations.
+  get              Print resources
+  help             describe subcommands and their syntax
+  kill             Stop the job. -rm will remove the job from history.
+  logs             Print logs of the job.
+  registry         Add registry secret on paddlecloud.
+  submit           Submit job to PaddlePaddle Cloud.
+
+Subcommands for PFS:
+  cp               upload or download files
+  ls               List files on PaddlePaddle Cloud
+  mkdir            mkdir directoies on PaddlePaddle Cloud
+  rm               rm files on PaddlePaddle Cloud
+
+
+Use "paddlectl flags" for a list of top-level flags
+```
+
+## Download the Demo Code and Try to Submit it
+
+We prepare some demo code to help users understanding how to submit
+a distributed training job to PaddleCloud, these demo codes are based
+on [Paddle Book](https://github.com/Paddlepaddle/book), you can find the
+tutorials for each chapter.
+
+You can fetch the demo code and submit the job as the following command:
+
+```bash
+> mkdir fit_a_line
+> cd fit_a_line
+> wget https://raw.githubusercontent.com/PaddlePaddle/cloud/develop/demo/fit_a_line/train.py
+> cd ..
+> paddlecloud submit -jobname fit-a-line -cpu 1 -gpu 1 -parallelism 1 -entry "python train.py train" fit_a_line/
+```
+
+For each submits arguments:
+
+- `-jobname`, STRING, the job name, you should specify a unique name.
+- `-cpu`, INT, CPU cores for each trainer node.
+- `-gpu`, INT, GPU cards for each trainer node,
+  if the cluster doesn't support GPU, please set `-gpu 0`.
+- `-parallelism`, INT, the parallelism, means trainer node count.
+- `-entry`, STRING, the entry point for the training job.
+- `./fit_a_line`, STRING, the local directory including job package.
+
+**NOTE1**: You can find the whole usage by `paddlectl submit -h`.
+**NOTE2**: Submit the job by different jobname, so that the previous jobs data would be saved on the cloud.
```

Comment:
"so that the previous jobs data would be saved on the cloud." => "so that it does not conflict with previous job with the same name"